src/FixedDataTableRow.js

```diff
@@ -201,17 +221,29 @@ class FixedDataTableRowImpl extends React.Component {
     var style = {
       width: this.props.width,
       height: this.props.height + subRowHeight,
+      zIndex: this.props.zIndex ? this.props.zIndex : 0,
+      position: 'absolute',
     };
```

Comment:
To differentiate the header or group header from the other rows we are using zIndex here .We will provide the styling for the row in this div only .